Scripts/Chapter.js

```diff
@@ -151,5 +153,25 @@ function FK_ToggleMinWidth(disable) {
  * @return {boolean} True if image is doubled (width >= height) false otherwise
  */
 function FK_IsDoublePage(img) {
+	if (FK_IsRightWidth(img)){
+		return false;
+	}
 	return (img.naturalWidth >= img.naturalHeight);
```

Comment:
As the return is a boolean, you don't need a if statement here. You can shorten the process by removing the condition and directly writing
return (!FK_IsRightWidth(img) || img.naturalWidth >= img.naturalHeight);

Reply:
That will not function (without adding a bunch of negations): the page might be wider than it is tall but still be the correct width (this is what I'm trying to fix), and therefore not need to be resized. If the page is already the correct width, then there is no need to check the ratio, it can simply return false to not be resized.

Reply:
You seem to have misread my code, haha. Your code and mine do exactly the same thing, I just compressed it in one line. If FK_IsRightWidth return true, then "return (!FK_IsRightWidth(img) || img.naturalWidth >= img.naturalHeight);" will return false no matter what. If FK_IsRightWidth return false, then the return continue with the || and return img.naturalWidth >= img.naturalHeight.
Same thing, but shorter (and probably a bit harder to understand ;))

Reply:
I had tried it and it failed to work, hence I went back and reasoned about it a bit more before undoing that change:
Your solution: Is it the right width? -> (yes: check ratio; can be true or false) (no: return true)
Mine: Is it the right width? -> (yes; return false) (no: check ratio)

Reply:
Flipping the checks around results in the resizing not functioning at all, so other changes would need to be made. I'm not sure it's worth it just to make it be one line.
A ternary or two might work to compact it down while still working though, like:
`return (FK_IsRightWidth ? false : (img.naturalWidth >= img.naturalHeight));`

Reply:
Wooops, you are right, I meant && not ||, sorry, I messed up.
"return (!FK_IsRightWidth(img) && img.naturalWidth >= img.naturalHeight);" is doing what you want. Here is a little demo code if you want to check https://pastebin.com/A6XFBc6F. You can run it and change the return values of the functions to see the different results.

Reply:
But you know what, if such a little detail is creating such a big mess, maybe it's for the best to leave the uncompressed version which is easier to understand xD